app/src/main/java/io/github/mattpvaughn/chronicle/features/player/NotificationBuilder.kt

```diff
@@ -63,12 +63,15 @@ class NotificationBuilder @Inject constructor(
         MediaButtonReceiver.buildMediaButtonPendingIntent(context, ACTION_PAUSE)
     )
     private val skipForwardsAction = NotificationCompat.Action(
-        R.drawable.ic_forward_30_white,
+        R.drawable.ic_forward_white,
+        // TODO: Access prefsRepo.jumpBackwardSeconds for custom icon
```

Comment:
you can access prefsRepo by injecting the prefsRepo in the constructor at the top of the file. it'd look like this:

```
class NotificationBuilder @Inject constructor(
    private val context: Context,
    private val plexConfig: PlexConfig,
    private val controller: MediaControllerCompat,
    private val currentlyPlaying: CurrentlyPlaying,
    private val prefsRepo: PrefsRepo
) {
```